barretenberg/cpp/src/barretenberg/benchmark/pippenger_bench/main.cpp

```diff
@@ -72,7 +72,9 @@ int pippenger()
     scalar_multiplication::pippenger_runtime_state<curve::BN254> state(NUM_POINTS);
     std::chrono::steady_clock::time_point time_start = std::chrono::steady_clock::now();
     g1::element result = scalar_multiplication::pippenger_unsafe<curve::BN254>(
-        { &scalars[0], /*size*/ NUM_POINTS }, reference_string->get_monomial_points(), state);
+        PolynomialSpan<const curve::BN254::ScalarField>{ /*start_index*/ 0, { &scalars[0], /*size*/ NUM_POINTS } },
```

Comment:
Noting ideally this explicit constructor be needed

Reply:
what do you mean here? like explicitly have this everywhere?

Reply:
Wouldn't be needed, sorry, having a constructor of polynomialspan just take span

Reply:
yeah, certainly an option, but I just opted to be more explicit about what we're passing in to pippenger for now